docs/src/blockchain-development/storage.md

```diff
@@ -6,6 +6,42 @@ Put in conventional programming terms, contract storage is like saving data to a
 
 Some basic use cases of storage include declaring an owner address for a contract and saving balances in a wallet.
 
+## Manual Storage Management
+
+Outside of the newer experimental `storage` syntax which is being stabalized, you can leverage FuelVM storage operations using the `store` and `get` methods provided in the standard library (`std`). Which currently works with primitive types.
+
+With this approach you will have to manually assign the internal key used for storage.
+
+An example is as follows:
+
+```sway
```

Comment:
Last thing is putting this in the `example/` directory as an example!

Reply:
@adlerjohn yep, that's last on my list, almost there.